CommonLibSF/include/SFSE/API.h

```diff
@@ -19,4 +19,29 @@ namespace SFSE
 
 	Trampoline& GetTrampoline();
 	void        AllocTrampoline(std::size_t a_size, bool a_trySFSEReserve = true);
+
+	namespace stl
+	{
+		template <typename T>
+		constexpr void write_thunk_call() noexcept
+		{
+			SFSE::AllocTrampoline(14);
+			auto& trampoline{ SFSE::GetTrampoline() };
+			T::func = trampoline.write_call<5>(T::address, T::Thunk);
+		}
+
+		template <typename TDest, typename TSource>
+		constexpr void write_vfunc() noexcept
+		{
+			REL::Relocation vtbl{ TDest::VTABLE[0] };
```

Comment:
Needs variant of this for classes using multiple VTABLES

Reply:
That's the one right below it, I copied that over from my CLibNG template which uses `REL::Variant`, I changed it to `REL::Offset` here since the former doesn't exist in CLibSF

Reply:
I mean something along the lines of `REL::Relocation vtbl{ TDest::VTABLE[idx] };`